apps/bc-wallet-api-server/src/controllers/AssetController.ts

```diff
@@ -57,6 +71,7 @@ class AssetController {
     }
   }
 
+  @Authorized()
```

Comment:
For all @Authorized annotations:
Don't we need to specify which roles are allowed to execute the endpoint?
i.e.
```
@Authorized('admin')
```

Reply:
Already asked @ronal2do about it. We are still not sure what the roles are for each endpoint.